apps/prairielearn/src/tests/helperServer.ts

```diff
@@ -67,7 +67,9 @@ export function before(courseDir: string | string[] = TEST_COURSE_PATH): () => P
       load.initEstimator('python', 1);
 
       debug('before(): initialize code callers');
-      await codeCaller.init();
+      await codeCaller.init({
+        lazyWorkers: true,
+      });
```

Comment:
Nit, this can fit on one line:

```suggestion
      await codeCaller.init({ lazyWorkers: true });
```